mlflow/shap.py

```diff
@@ -179,3 +255,329 @@ def predict_function(X) -> pred:
+                      :func:`get_default_conda_env()` environment is added to the model.
+                      The following is an *example* dictionary representation of a Conda
+                      environment::
+
+                        {
+                            'name': 'mlflow-env',
+                            'channels': ['defaults'],
+                            'dependencies': [
+                                'python=3.6.0',
+                                'shap=0.37.0'
+                            ]
+                        }
+    :param registered_model_name: (Experimental) If given, create a model version under
+                                  ``registered_model_name``, also creating a registered model if one
+                                  with the given name does not exist.
+
+    :param signature: (Experimental) :py:class:`ModelSignature <mlflow.models.ModelSignature>`
+                      describes model input and output :py:class:`Schema <mlflow.types.Schema>`.
+                      The model signature can be :py:func:`inferred <mlflow.models.infer_signature>`
+                      from datasets with valid model input (e.g. the training dataset with target
+                      column omitted) and valid model output (e.g. model predictions generated on
+                      the training dataset), for example:
+
+                      .. code-block:: python
+
+                        from mlflow.models.signature import infer_signature
+                        train = df.drop_column("target_label")
+                        predictions = ... # compute model predictions
+                        signature = infer_signature(train, predictions)
+    :param input_example: (Experimental) Input example provides one or several instances of valid
+                          model input. The example can be used as a hint of what data to feed the
+                          model. The given example will be converted to a Pandas DataFrame and then
+                          serialized to json using the Pandas split-oriented format. Bytes are
+                          base64-encoded.
+    :param await_registration_for: Number of seconds to wait for the model version to finish
+                            being created and is in ``READY`` status. By default, the function
+                            waits for five minutes. Specify 0 or None to skip waiting.
+
+    """
+
+    Model.log(
+        artifact_path=artifact_path,
+        flavor=mlflow.shap,
+        explainer=explainer,
+        conda_env=conda_env,
+        serialize_model_using_mlflow=serialize_model_using_mlflow,
+        registered_model_name=registered_model_name,
+        signature=signature,
+        input_example=input_example,
+        await_registration_for=await_registration_for,
+    )
+
+
+@experimental
+def save_model(
+    explainer,
+    path,
+    serialize_model_using_mlflow=True,
```

Comment:
Can we mention that `serialize_model_using_mlflow` currently only supports `sklearn` and `pytorch` flavors?

Reply:
Looks like the current docstring doesn't contain a parameter description for `serialize_model_using_mlflow`. Can you add?

Reply:
Added docstring and mentioned only sklearn and pytorch are supported for this method or serialization